### PR TITLE
docs(nm): added documentation for vlan and mtu configuration

### PR DESCRIPTION
**Pull request by fdizazzo**
> **Note**: We are using the Conventional Commits convention for our pull request titles. Please take a look at the [PR title format document](https://github.com/eclipse/kura/blob/develop/CONTRIBUTING.md#submitting-the-changes) for the supported [types](https://github.com/eclipse/kura/blob/develop/CONTRIBUTING.md#type) and [scopes](https://github.com/eclipse/kura/blob/develop/CONTRIBUTING.md#scope).

This PR adds documentation for #4877 and #4816

### CODE REVIEW

**Review by mattdibi — COMMENTED** [review]
LGTM

Only one thing: we think that the addition of the support for VLANs is so important that requires a dedicated page in the Network Configuration section on its own, like we did for "_Network Failover_" and "_Wi-Fi 802.1x Configuration_". Like this:

image

Obviously for now it will be a barebone page since we still don't have a UI, but will be updated in the future when we do.

The page should contain some basic info about what VLANs are, how Kura treats them, some further explaination of what parameters are required etc.

Do you think you can handle that?

**Comment by fdizazzo** [comment]
Sure, but i'll need a few days :)

**Comment by mattdibi** [comment]
@fdizazzo no problemo! 👍

File: docs/gateway-configuration/vlan-configuration.md
Comment (mattdibi) [inline docs/gateway-configuration/vlan-configuration.md]:
```suggestion
For devices configured to use [NetworkManager](https://networkmanager.dev), it is possible to configure multiple VLAN interfaces. Take a look at our [installer profiles](../getting-started/install-kura.md#installer-types) to know which are supported.
```

File: docs/gateway-configuration/vlan-configuration.md
Comment (mattdibi) [inline docs/gateway-configuration/vlan-configuration.md]: Formatting

```suggestion
    When creating a new VLAN be sure to include the `net.interfaces` parameter, containing both the previously existing network interface and the name of the new vlan to be created.
```

File: docs/gateway-configuration/vlan-configuration.md
Comment (mattdibi) [inline docs/gateway-configuration/vlan-configuration.md]:
```suggestion
Once a VLAN is created it can be managed via the Kura UI just like any other Ethernet interface.
```

File: docs/gateway-configuration/vlan-configuration.md
Comment (mattdibi) [inline docs/gateway-configuration/vlan-configuration.md]: Not only it deletes the configuration but the actual virtual interface. I would document that explicitly

```suggestion
    Setting a VLAN status to Disabled deletes its configuration in NetworkManager and the virtual interface from the system. Althougt it will no longer be visible on the UI, all the configurations are left in Kura. Therefore the VLAN can be restored by setting the `net.interface.<interface>.config.ip4.status` to `netIPv4StatusEnabledLAN` or `netIPv4StatusEnabledWAN` via snapshot upload, then resume configuration via UI.
```

File: docs/gateway-configuration/vlan-configuration.md
Comment (mattdibi) [inline docs/gateway-configuration/vlan-configuration.md]: Wrong reference resulting in broken image

image


```suggestion

![VLAN UI management](./images/vlan-interface-example.png)
```

**Review by mattdibi — CHANGES_REQUESTED** [review]
Some minor issues, otherwise LGTM 👍  Once the issues are fixed I'll proceed with merging.

GJ @fdizazzo 😄

**Comment by fdizazzo** [comment]
... and it's done 😃

File: docs/gateway-configuration/vlan-configuration.md
Comment (mattdibi) [inline docs/gateway-configuration/vlan-configuration.md]: Spacing

```suggestion
Once a VLAN is created it can be managed via the Kura UI just like any other Ethernet interface.

![VLAN UI management](./images/vlan-interface-example.png)
```

File: docs/gateway-configuration/vlan-configuration.md
Comment (mattdibi) [inline docs/gateway-configuration/vlan-configuration.md]:
```suggestion
    Setting a VLAN status to "_Disabled_" deletes its configuration in NetworkManager and the related virtual interface from the system. Although it will is no longer be visible on the UI, all the configurations are left in Kura. Therefore the VLAN can be restored by setting the `net.interface.<interface>.config.ip4.status` to `netIPv4StatusEnabledLAN` or `netIPv4StatusEnabledWAN` via snapshot upload, then resume configuration via UI.
```

**Review by mattdibi — APPROVED** [review]

**Comment by mattdibi** [comment]
@fdizazzo thanks for your contribution 😄

**Pull request merged**
